pkg/api/customization/monitor/handler_utils.go

```diff
@@ -63,40 +80,102 @@ func (p *clusterGraphInputParser) parseFilter() error {
 		return fmt.Errorf("clusterId is empty")
```

Comment:
Maybe the error should say something like "Filters is missing, it is a required input" -> message talks about clusterId, but is checking about Filters